man/git-alias.md

```diff
@@ -6,13 +6,23 @@ git-alias(1) -- Define, search and show aliases
 `git-alias`  
 `git-alias` &lt;search-pattern&gt;  
 `git-alias` &lt;alias-name&gt; &lt;command&gt;  
+`git-alias` [--global] &lt;alias-name&gt; &lt;command&gt;
+`git-alias` [--local] &lt;alias-name&gt; &lt;command&gt;
 
 ## DESCRIPTION
 
   List all aliases, show one alias, or set one (global) alias.
 
 ## OPTIONS
 
+  --global
+
+  Show aliased matched in the system config
```

Comment:
Let's document that we can also use the option to alias a command in different scope.

Reply:
Mean to give an example?

Reply:
```suggestion
  Show or create alias in the system config
```